### PR TITLE
Host verify connection

### PR DESCRIPTION
**Pull request by ineiti**
Handle connections better:
- timeout to prune stale connections
- if a write fails, try to re-open it

### CODE REVIEW

**Comment by ineiti** [comment]
This is just in the case a `SendRaw` fails to re-open the connection and to try again.

File: lib/sda/export_test.go [hunk 1]
```diff
@@ -47,3 +47,21 @@ func (o *Overlay) TokenToNode(tok *Token) (*TreeNodeInstance, bool) {
...
+
+func (h *Host) AbortConnections() error {
+	for _, c := range h.connections {
+		h.unregisterConnection(c)
+	}
+	close(h.ProcessMessagesQuit)
+	return h.host.Close()
+}
+
+func (h *Host) CloseConnections() error {
```
Comment (nikkolasg) [inline lib/sda/export_test.go]: CloseConnection should it not unregister the connection too ? Since it's closed we won't use it anymore ..

Comment (ineiti) [inline lib/sda/export_test.go]: Line 53 is not what you're looking for?

Comment (ineiti) [inline lib/sda/export_test.go]: Sorry - `CloseConnection` initialises the `connections`-map anyway.

**Comment by ineiti** [comment]
Superseeded by #395

**Pull request closed without merging**